lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
+    attribute :priority,        Integer, default: 0
+    attribute :unit,            String
+    attribute :user_value,      Float
+
+    validates_presence_of :query
+
+    def self.collection_from_hash(init_hash)
+      init_hash.map do |key, user_value|
+        input = find(key)
+        input.user_value = user_value
```

Comment:
Ok, now I see why you want this to be part of the `InitializerInput`. But since we do not do it like this for `Input`, we should not do it here, either. In any way, the `user_value` must not be an _attribute_, it could at most be a plain old instance variable.